generate.py

```diff
@@ -384,7 +384,8 @@ def generate(args):
             formatted_prompt = args.prompt.replace(" ", "_").replace("/",
                                                                      "_")[:50]
             suffix = '.png' if "t2i" in args.task else '.mp4'
-            args.save_file = f"{args.task}_{args.size}_{args.ulysses_size}_{args.ring_size}_{formatted_prompt}_{formatted_time}" + suffix
+            safe_size = args.size.replace("*", "x")
```

Comment:
lets do this only when windows sys is detected...to be really safe and not break any existing (linux) pipeline.

Reply:
> lets do this only when windows sys is detected...to be really safe and not break any existing (linux) pipeline.

Yes, I have revised it again, what do you think?